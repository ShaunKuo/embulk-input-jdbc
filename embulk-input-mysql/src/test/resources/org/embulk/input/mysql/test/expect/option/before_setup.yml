table: test2
order_by: 'id'
before_setup: create table test2 (id int, num int); insert into test2 values(1, 999), (2, 0);
d: insert into test2 values(1, 999), (2, 0);
options:
  allowMultiQueries: true