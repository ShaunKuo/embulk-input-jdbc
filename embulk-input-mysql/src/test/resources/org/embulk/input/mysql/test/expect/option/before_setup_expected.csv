1,999
2,0
